base/Dockerfile

```diff
@@ -42,7 +42,10 @@ RUN add-apt-repository -y ppa:git-core/ppa \
 # '-l': see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user
 RUN useradd -l -u 33333 -G sudo -md /home/gitpod -s /bin/bash -p gitpod gitpod \
     # passwordless sudo for users in the 'sudo' group
-    && sed -i.bkp -e 's/%sudo\s\+ALL=(ALL\(:ALL\)\?)\s\+ALL/%sudo ALL=NOPASSWD:ALL/g' /etc/sudoers
+    && sed -i.bkp -e 's/%sudo\s\+ALL=(ALL\(:ALL\)\?)\s\+ALL/%sudo ALL=NOPASSWD:ALL/g' /etc/sudoers \
+    # To emulate the workspace-session behavior within dazzle build env
+    && mkdir /workspace && chown -hR gitpod:gitpod /workspace
```

Comment:
@princerachit I'm not sure if this change is acceptable for the base image. Can I ask you to look at this PR?

Reply:
It looked fine to me, but would like to hear what @princerachit has to say.

Reply:
I did this on the base image because this was already necessary for another chunk (`tool-postgresql`), so didn't want to duplicate the same thing over different chunks.